source/backend/api/Models/Concepts/Note/ProjectNoteMap.cs

```diff
@@ -0,0 +1,27 @@
+using Mapster;
+using Entity = Pims.Dal.Entities;
+
+namespace Pims.Api.Models.Concepts
+{
+    public class ProjectNoteMap : IRegister
+    {
+        public void Register(TypeAdapterConfig config)
+        {
+            config.NewConfig<Entity.PimsProjectNote, EntityNoteModel>()
```

Comment:
instead of creating a new file for the config the EntityNoteModel could do the mapping like the other ones